Cargo.toml

```diff
@@ -15,3 +15,6 @@ exclude = [".gitignore"]
 
 [dev-dependencies]
 clap = { version = "4.5.3", features = ["derive"] }
+
+[dependencies]
+unicode-bidi = "0.3.15"
```

Comment:
[Q] 
1. Is using external crates OK?
2. If using crates is possible, is it better to keep them in `dev-dependencies` instead?

Reply:
1. Yes, it's okay to have deps. Ideally, they'd be `#[no_std]` like the rest of the lib, but if they're not, we can just put this change behind a feature.
2. `dev-dependencies` are for the `build.rs`, tests, and examples! The actual deps of the library would go under `dependencies`, except there are none yet! :)